### PR TITLE
Run integration tests from `make test`

### PR DESCRIPTION
**Pull request by tsg**
Also added a `unit` target that runs only the unit tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,10 +26,14 @@ clean:
 run: filebeat
 	./filebeat -c etc/filebeat.dev.yml -e -v -d "*"
 
-.PHONY: test
-test:
+.PHONY: unit
+unit:
 	$(GODEP) go test -short ./...
```
Comment (ruflin) [inline Makefile]: should we also run the "long" tests here? I'm asking because I would except `make test` to run the full suite.

Comment (tsg) [inline Makefile]: For the Beats it currently doesn't matter as they have no "long" tests. For libbeat, that would mean that you need Elasticsearch and Redis when running `make test`. I'm not sure about that part.

Comment (ruflin) [inline Makefile]: Jep, which we have with the docker environment. As now everyone can run these tests without setup, they should also be run locally before pushing.

**Comment by tsg** [comment]
@ruflin Removed the `-short` param.

**Pull request merged**